apps/web/src/components/Dropzone/styles.ts

```diff
@@ -1,16 +1,74 @@
 import { styled } from 'stitches/stitches.config';
 
+export const Overlay = styled('div', {
```

Comment:
move these styled to the new `AvatarDropzone` component

Reply:
done